notebooks/demo/mg_property_graph.ipynb

```diff
@@ -0,0 +1,217 @@
+    "from dask.distributed import Client, wait\n",
+    "from dask_cuda import LocalCUDACluster\n",
+    "from cugraph.dask.comms import comms as Comms\n",
+    "import cugraph.dask as dask_cugraph\n",
+    "import cugraph\n",
+    "import dask_cudf\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import urllib.request\n",
+    "import os\n",
+    "\n",
+    "data_dir = '../data/'\n",
+    "if not os.path.exists(data_dir):\n",
+    "    print('creating data directory')\n",
+    "    os.system('mkdir ../data')\n",
+    "\n",
+    "# download the Hollywood dataset\n",
+    "base_url = 'https://rapidsai-data.s3.us-east-2.amazonaws.com/cugraph/benchmark/'\n",
+    "fn = 'hollywood.csv'\n",
+    "comp = '.gz'\n",
+    "\n",
+    "if not os.path.isfile(data_dir+fn):\n",
+    "    if not os.path.isfile(data_dir+fn+comp):\n",
+    "        print(f'Downloading {base_url+fn+comp} to {data_dir+fn+comp}')\n",
+    "        urllib.request.urlretrieve(base_url+fn+comp, data_dir+fn+comp)\n",
+    "    print(f'Decompressing {data_dir+fn+comp}...')\n",
+    "    os.system('gunzip '+data_dir+fn+comp)\n",
+    "    print(f'{data_dir+fn+comp} decompressed!')\n",
+    "else:\n",
+    "    print(f'Your data file, {data_dir+fn}, already exists')\n",
+    "\n",
+    "# File path, assuming Notebook directory\n",
+    "input_data_path = data_dir+fn"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Initialize multi-GPU environment\n",
+    "Before we get started, we need to setup a Dask local cluster of workers to execute our work and a client to coordinate and schedule work for that cluster. As we see below, we can initiate a cluster and client using only 3 lines of code."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cluster = LocalCUDACluster()\n",
+    "client = Client(cluster)\n",
+    "Comms.initialize(p2p=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Read the data from disk\n",
+    "cuGraph depends on cudf for data loading and the initial DataFrame creation. The CSV data file contains an edge list, which represents the connection of a vertex to another. The source to destination pairs is what is known as Coordinate Format (COO). In this test case, the data is just two columns. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Start ETL timer\n",
+    "t_start = time.time()\n",
+    "\n",
+    "# Helper function to set the reader chunk size to automatically get one partition per GPU  \n",
+    "chunksize = dask_cugraph.get_chunksize(input_data_path)\n",
```

Comment:
I wonder if we should briefly explain what chunk size is, what a partition is, and why we'd want 1 partition per GPU, or are we assuming the reader already knows or doesn't need to know?